data/src/main/java/io/gnosis/data/models/TransferDetails.kt

```diff
@@ -0,0 +1,61 @@
+package io.gnosis.data.models
+
+import io.gnosis.data.backend.dto.DetailedExecutionInfo
+import io.gnosis.data.backend.dto.TxData
+import pm.gnosis.model.Solidity
+import java.util.*
+
+//FIXME: (WIP) revise structure
+// Designs:
+// incoming Transfer https://zpl.io/V1KKZq5 (with data section)
```

Comment:
I don't think we should comments with links to design specs in code. And also we should not have comments about design specs at all, just code related comments

Reply:
Yes you're right.
Those were work in progress comments. While I was checking the data from the client gateway has everything we need in the layouts. I wanted to remove them when the layouts are done. But if you want, I can remove them right now.

Reply:
sure, when layouts are done, let's clean it up